components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/modules/business-controllers/device.js

```diff
@@ -300,7 +306,9 @@ deviceModule = function () {
 			);
 		} else {
 			log.error("User object was not found in the session");
-			throw constants["ERRORS"]["USER_NOT_FOUND"];
+            userModule.logout(function () {
```

Comment:
Fix formatting issue.

Reply:
Fixed in https://github.com/wso2/carbon-device-mgt/pull/1275